packages/remote-feature-flag-controller/src/utils/version-utils.test.ts

```diff
@@ -0,0 +1,198 @@
+import {
+  isVersionAtLeast,
+  isMultiVersionFeatureFlagValue,
+  selectVersionFromMultiVersionFlag,
+} from './version-utils';
+
+describe('isVersionAtLeast', () => {
```

Comment:
Minor, should we be exporting this just to test it? Are we just considering it high risk?